pxr/usd/usdLux/overview.dox

```diff
@@ -101,11 +101,54 @@ constraints, such as to attach lights to moving geometry.  This is
 consistent with USD's policy of storing the computed result of rigging,
 not the rigging itself.
 
+\subsection usdLux_quantities Quantities and Units
+
+Most renderers consuming OpenUSD today are RGB renderers, rather than spectral.
+Units in RGB renderers are tricky to define as each of the red, green and blue
+channels transported by the renderer represents the convolution of a spectral
+exposure distribution, e.g. CIE Illuminant D65, with a sensor response function, e.g.
+CIE 1931 𝓍̅. Thus the main quantity in an RGB renderer is neither radiance nor
+luminance, but "integrated radiance" or "tristimulus weight".
+
+The emision of a default light with `intensity` 1 and `color` [1, 1, 1] is an
```

Comment:
since this section is copy/paste of things I already noted above, eg, "emision" typo and subsequent notes, this comment is merely a note to propagate fixes here as well.

Reply:
Yup, done!

https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-3042b10f5eed7d1dc28467437ef7f7c3cad9ccb30eb8dd93820575402328c11eR113